shared/chat/conversation/info-panel/attachments.tsx

```diff
@@ -527,14 +559,16 @@ export default (p: Props) => {
               l.push({
                 author: m.author,
                 ctime: m.timestamp,
+                key: `unfurl-empty-${m.ordinal}-${m.author}-${m.timestamp}`,
                 snippet: (m.decoratedText && m.decoratedText.stringValue()) ?? '',
               })
             } else {
-              ;[...m.unfurls.values()].forEach(u => {
+              ;[...m.unfurls.values()].forEach((u, i) => {
                 if (u.unfurl.unfurlType === RPCChatTypes.UnfurlType.generic && u.unfurl.generic) {
                   l.push({
                     author: m.author,
                     ctime: m.timestamp,
+                    key: `unfurl-${m.ordinal}-${i}-${m.author}-${m.timestamp}-${u.unfurl.generic.url}`,
```

Comment:
shouldn't the ordinal be enough to uniquely id this?

Reply:
A single message can have multiple links which are unfurled. So multiple `m.unfurls.values()` sharing an ordinal. Ordinal+i should be sufficient. I have not tested to see if this reasoning is reflected irl.